.github/workflows/release.yaml

```diff
@@ -1,32 +1,78 @@
 name: Release
 
 on:
+  schedule:
+    # Run on every first day of the month
+    - cron: '0 0 1 * *'
   push:
     tags:
       - '**'
 
 jobs:
 
-  release:
+  changelog:
     runs-on: ubuntu-latest
     steps:
-
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
-      - name: Create Package
-        run: tar -czf forgit-${{github.ref_name}}.tar.gz --exclude LICENSE --exclude README.md *
-
       - name: Generate Changelog
         id: changelog
         run: .github/scripts/changelog.sh > CHANGELOG.md
+        # When the script returns with an error, it indicates that we don't have
+        # any public changes to be released. This is expected behavior, so we
+        # quit the workflow successfully in this case.
         continue-on-error: true
```

Comment:
The comment seems at odds with the message here 

> This is expected behavior, so we quit the workflow successfully in this case.

Wouldn't this imply `continue-on-error: false`?

Reply:
I think this is fine. This ensures that the job does not fail ungracefully (sending out notifications to us etc.). Since the following steps all use `if: steps.changelog.outcome == 'success'` they're simply skipped over and the workflow ends successfully without doing any further steps.

Reply:
What @sandr01d says. `continue-on-error: false` is the default behavior and would quit the workflow with an error, which is not what we want.